web/frontend/src/components/slam.vue

```diff
@@ -241,6 +322,11 @@ const toggle3dExpand = () => {
   updateSLAM3dRefreshFrequency(props.name, selected3dValue);
 };
 
+const toggle2dExpand = () => {
```

Comment:
I don't think this has been properly rebased on main. I think this will break some fixes which have gone in recently.

Reply:
fixed now.